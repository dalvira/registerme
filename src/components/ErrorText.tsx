import React from 'react';
import {Text, StyleSheet} from 'react-native';
import {colors} from '../theme/colors';

interface Props {
  testID?: string;
  text: string;
}

const ErrorText = (props: Props) => {
  return (
    <Text testID={props.testID} style={styles.text}>
      {props.text}
    </Text>
  );
};

export default ErrorText;

const styles = StyleSheet.create({
  text: {
    color: colors.red,
    fontSize: 16,
    fontWeight: '600',
    marginBottom: 20,
  },
});
