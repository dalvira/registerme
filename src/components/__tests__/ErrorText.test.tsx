import React from 'react';
import renderer from 'react-test-renderer';
import ErrorText from '../ErrorText';

it('renders correctly', () => {
  const tree = renderer.create(<ErrorText text="Error" />).toJSON();
  expect(tree).toMatchSnapshot();
});
