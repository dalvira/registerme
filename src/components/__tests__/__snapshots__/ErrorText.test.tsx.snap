// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "#e8193f",
      "fontSize": 16,
      "fontWeight": "600",
      "marginBottom": 20,
    }
  }
>
  Error
</Text>
`;
